board/recalbox/fsoverlay/recalbox/share_init/system/.emulationstation/es_systems.cfg

```diff
@@ -67,6 +67,7 @@
 		    <core>gliden64</core>
 		    <core>n64</core>
 		    <core>rice</core>
+    		    <core>glide64mk2</core>
```

Comment:
or,
by board, have a n64_emu, for example for the xu4

<core>glide64mk2</core>

and sed a keyword like <N64_INCLUDES>

by the one of the file in a post script.

and check that no <[A-Z]*_INCLUDES> are remaining



2016-06-04 9:22 GMT+02:00 Subs <notifications@github.com>:

> In
> board/recalbox/fsoverlay/recalbox/share_init/system/.emulationstation/es_systems.cfg
> <https://github.com/recalbox/recalbox-buildroot/pull/367#discussion_r65798836>
> :
>
> > @@ -67,6 +67,7 @@
> >  		    <core>gliden64</core>
> >  		    <core>n64</core>
> >  		    <core>rice</core>
> > +    		    <core>glide64mk2</core>
>
> Add a post install sed for xu4 only that adds the core ?
> It's the first platform that is not a pi, so maybe we should consider
> handling some files like the recalbox.conf afterall : per platform
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/recalbox/recalbox-buildroot/pull/367/files/84cbb069e17dde9c1972a2a7f7d3d267c25b0b53#r65798836>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFGWYXcPa7T6ZU6Dim2H9XiAn5mzvePZks5qISexgaJpZM4Is1I6>
> .
>